en/docs/developer-portal/applications/sdks/java-android.md

```diff
@@ -18,8 +18,8 @@ implementation 'org.wso2.identity.sdk.android.oidc:wso2-oidc-sdk:0.0.5'
 
 ### Add a URI Scheme 
   
-You need to add a redirect scheme in the application. You need to add the `appAuthRedirectScheme` in your app's
- `build.gradle` file.
+You need to add a redirect scheme in the application. You need to add the `appAuthRedirectScheme` in your application
```

Comment:
You need to add `appAuthRedirectScheme` in your application level `build.gradle` file.